recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,15 @@
+numpy:
+- 1.16 # [not (aarch64 or s390x or arm64)]
+- 1.16 # [not (aarch64 or s390x or arm64)]
+- 1.19 # [aarch64 or s390x]
+- 1.19 # [aarch64 or s390x or arm64]
+- 1.19
```

Comment:
Statsmodels was built with a forced constraint for numpy for at least some of our platforms:

https://github.com/AnacondaRecipes/statsmodels-feedstock/blob/master/recipe/meta.yaml#L30

As a result our 1.16 pins do not work. I modified our pins as minimally as possible here so that this package can be installed. I don't see this as a gigantic problem but trust me, this package cannot be built on 1.16 on the platforms I've overridden above. I tried!

Reply:
so this means on python 3.9, just version 1.19 of numpy can be used here.  Fair enough

Reply:
Yes, indeed. That's the basic effect. And of course it is just a lower bound